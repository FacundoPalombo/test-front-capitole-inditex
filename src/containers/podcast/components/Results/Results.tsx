import { number } from 'prop-types';
import React from 'react';

import styles from './styles.module.scss';

type ResultsProps = {
  results: number;
};

const Results = ({ results }: ResultsProps) => {
  return (
    <div className={styles.results__card}>
      <h2 className={styles.results__title}>Episodes: {results}</h2>
    </div>
  );
};

Results.propTypes = {
  results: number,
};

export default Results;
