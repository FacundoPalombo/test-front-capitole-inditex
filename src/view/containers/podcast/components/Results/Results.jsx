import { number } from 'prop-types';
import React from 'react';

import styles from './styles.module.scss';

const Results = ({ results }) => {
  return (
    <div className={styles.results__card}>
      <h2 className={styles.results__title}>Total tracks: {results}</h2>
    </div>
  );
};

Results.propTypes = {
  results: number,
};

export default Results;
