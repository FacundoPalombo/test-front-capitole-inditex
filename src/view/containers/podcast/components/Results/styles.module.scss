.results {
  &__card {
    box-shadow: 0px 1px 4px 1px var(--shadow);
    display: flex;
    align-items: flex-start;
    justify-content: center;
    flex-direction: column;
    max-height: 800px;
    padding: 8px;
    overflow: visible;
    margin-top: auto;
    border-radius: 4px;
  }
  &__title {
    color: var(--text-primary);
    font-weight: 700;
  }
}
