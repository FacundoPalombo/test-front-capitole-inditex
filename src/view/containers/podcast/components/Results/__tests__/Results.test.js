import React from 'react';
import { render } from '@testing-library/react';
import Results from '../Results';

describe('Results unit tests', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Results />);
    expect(asFragment()).toMatchSnapshot();
  });
});
