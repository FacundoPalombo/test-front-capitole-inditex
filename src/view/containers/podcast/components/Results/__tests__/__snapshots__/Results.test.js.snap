// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results unit tests should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="results__card"
  >
    <h2
      class="results__title"
    >
      Episodes: 
    </h2>
  </div>
</DocumentFragment>
`;
